### DISCUSSION TITLE
cmake 3.22.0

### DISCUSSION THREAD

**Discussion by thuyumeng** [bazelbuild/rules_foreign_cc#903] [Q&A] [answered]
hi, Would you please support Cmake version 3.22.0?
Can I Implement it myself?

**Comment by jsharpe** [comment, marked as answer]
Its already supported: https://github.com/bazelbuild/rules_foreign_cc/blob/c57b55f2f6207c0c44c2b4a51dc235955d4f47b2/toolchains/prebuilt_toolchains.bzl#L280
Pass `cmake_version="3.22.0"` when you call `rules_foreign_cc_dependencies` to select this version of cmake.
